js/flightlog_parser.js

```diff
@@ -309,6 +310,20 @@ var FlightLogParser = function(logData) {
             vbat_sag_compensation: null,
             gyro_to_use: null,
             dynamic_idle_min_rpm: null,
+            ff_transition: null,
```

Comment:
Why we have two variables that are the same? This `ff_transition`and the one of line 223 `feedforward_transition`?

I think you are understanding bad how this works: you must have only **one** variable to store the value. With the name you want, and use the translation table to "translate" the content of the blackbox header to the internal name if it is not the same. In this way, if this parameter has different names between versions, we will have only one internal variable, and all the translations needed in the translations table to assign it correctly.

Reply:
It is like that because I do not know how to do it correctly.  I could not find an example in the translation table to follow.  And this worked, even if it was bad  :-)

If you could me exactly what I must do to fix this particular example, then I can do it correctly for the others.

Reply:
This part is easy, I suppose you want to use the "new" name (if not, do the inverse of what I say here). The process is like this:
1. Remove the old declaration: 
```javascript
feedforward_transition:null,            // Feedforward transition
```
2. Add the new declaration:
```javascript
ff_transition:null,            // Feedforward transition
```
2. Add the value to the `translationValues` table, to maintain backwards compatibility:
```javascript
feedforward_transition : "ff_transition",
``` 
3. Replace all the references to the old `feedforward_transition` to the new `ff_transition` in the code.

Reply:
I have a working version of the log viewer where I can edit directly the .js files.
If I attempt these changes it will not open a log file. 
I am doing something wrong.  
I'll push a commit, but it seems to not work properly.

Reply:
I've not tested it but seems ok :)

Reply:
I tested, works great!  Thanks for explaining how to do it properly.